### PR TITLE
Connecting Form to Prefill, switching the address component to use form schema

### PR DESCRIPTION
**Pull request by mojaray2k**
## Description
As a developer I need place the address component in the VA From JSON Schema so I can see the data from the form populate the review page.

## Testing done
Tested Api call from the backend which is called on  http://localhost:3001/hearing-aid-batteries-and-accessories/veteran-information/personal-details

## Screenshots
Cursor_and_Request_for_hearing_aid_batteries_and_accessories___Veterans_Affairs

Cursor_and_Request_for_hearing_aid_batteries_and_accessories___Veterans_Affairs

## Acceptance criteria
- [ ] Should be able to go to http://localhost:3001/hearing-aid-batteries-and-accessories/veteran-information/personal-details and see form prefill data with veterans personal information
- [ ] Should be able to go to http://localhost:3001/hearing-aid-batteries-and-accessories/veteran-information/addresses and see the form prefill data with

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/hca/helpers.jsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ export function prefillTransformer(pages, formData, metadata, state) {
     newData = { ...newData, 'view:isUserInMvi': true };
   }
 
+  // eslint-disable-next-line no-console
+  console.log(formData);
```
Comment (ncksllvn) [inline src/applications/hca/helpers.jsx]: Looks like some leftover debugging code

**Review by ncksllvn — APPROVED** [review]
Not a forms expert so my review is a little naive, but it LGTM 🙂

File: src/applications/disability-benefits/2346/prefillTransformer.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unreachable */
```
Comment (mr0sari0) [inline src/applications/disability-benefits/2346/prefillTransformer.js]: We can get rid of this line.

**Review by maharielrosario — APPROVED** [review]

File: src/applications/disability-benefits/2346/prefillTransformer.js (hunk 2 above)
Comment (mr0sari0) [inline src/applications/disability-benefits/2346/prefillTransformer.js]: We could get rid of this line as well.

File: src/applications/hca/helpers.jsx (hunk 1 above)
Comment (mojaray2k) [reply src/applications/hca/helpers.jsx]: Nice catch I just removed it.

File: src/applications/disability-benefits/2346/prefillTransformer.js (hunk 2 above)
Comment (mojaray2k) [reply src/applications/disability-benefits/2346/prefillTransformer.js]: Yeah I got rid of the file.

**Review by bkjohnson — APPROVED** [review]

**Comment by mojaray2k** [comment]
> Not a forms expert so my review is a little naive, but it LGTM 🙂

Thanks @ncksllvn  for the review

**Comment by mojaray2k** [comment]
@bkjohnson  Thank you for the review.

**Review by bkjohnson — DISMISSED** [review]
It looks like one of your changes [broke a unit test](https://app.circleci.com/jobs/github/department-of-veterans-affairs/vets-website/1760/tests).

**Comment by mojaray2k** [comment]
> It looks like one of your changes [broke a unit test](https://app.circleci.com/jobs/github/department-of-veterans-affairs/vets-website/1760/tests).
@bkjohnson 
What change? I thought I was supposed to add my form to the this file `src/platform/forms/tests/forms.unit.spec.js`

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**